cmd/juju/application/config_test.go

```diff
@@ -169,7 +169,19 @@ func (s *configCommandSuite) TestSetCommandInit(c *gc.C) {
 
 	// --reset and no config name provided
 	err = coretesting.InitCommand(application.NewConfigCommandForTest(s.fake), []string{"application", "--reset"})
-	c.Assert(err, gc.ErrorMatches, "no configuration options specified")
+	c.Assert(err, gc.ErrorMatches, "flag needs an argument: --reset")
```

Comment:
Line 110. I kept the existing test style here, which is a bit different than the other two places where this collapse is happening. Once it lands and it is decided we are going to keep it, we can make a pass at refactoring the three suites to be more congruous.